src/Symfony/Component/Translation/Bridge/Loco/LocoProvider.php

```diff
@@ -91,92 +84,137 @@ public function read(array $domains, array $locales): TranslatorBag
     {
         $domains = $domains ?: ['*'];
         $translatorBag = new TranslatorBag();
+        $responses = [];
 
         foreach ($locales as $locale) {
             foreach ($domains as $domain) {
-                $response = $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', $locale, $domain));
+                $responses[] = [
+                    'response' => $this->client->request('GET', sprintf('export/locale/%s.xlf?filter=%s&status=translated', $locale, $domain)),
```

Comment:
you should apply `urlencode` (or maybe `rawurlencode`, I never remember which one is appropriate in the path or in the query string) on both variables to be safe about weird chars

Reply:
I applied `rawurlencode`